pkg/apis/enricher/framework/dotnet/dotnet_detector.go

```diff
@@ -31,6 +31,11 @@ func (d DotNetDetector) GetSupportedFrameworks() []string {
 	return []string{""}
 }
 
+func (d DotNetDetector) GetApplicationFileInfos(componentPath string, ctx *context.Context) []model.ApplicationFileInfo {
+	// not implemented yet
+	return []model.ApplicationFileInfo{}
```

Comment:
Nitpicking, but I'd suggest returning `nil`, which is a valid slice of length 0 (and faster to type ^^)
```suggestion
	return nil
```

Reply:
I've updated all detectors where `GetApplicationFileInfos` is not implemented to return `nil`